src/test/java/org/cactoos/scalar/ScalarWithFallbackTest.java

```diff
@@ -58,6 +58,22 @@ public void usesMainFunc() throws Exception {
         );
     }
 
+    @Test
+    public void usesMainFuncInNonIterableCtor() {
+        final String message = "Main function's result";
+        MatcherAssert.assertThat(
+            "Scalar's main function value mismatch",
```

Comment:
@scratchboom WDYT change this message to something like "Can't match value of Scalar's main function"?